modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+`null` is not a valid value for `input` and, as a consequence, an exception was thrown.
+
+With this update, the `editor.selection.isForward()` API bug has been corrected, invalid selections no longer occur, a valid `input` value is passed to `input.parentNode.removeChild(input)`, and exceptions are no longer thrown.
+
+
+=== The InsertParagraph or `mceInsertNewLine` commands did not delete the current selection like the native command does
+
+The `insertParagraph` command was added to provide a {productname}-native replication of browser implementations of this functionality.
+
+However, when a string was selected and the `insertParagraph` was then executed, it did not delete the selected string and insert the paragraph break as expected. Instead, the command inserted a paragraph break at the beginning of the selected string and added the selection to the new trailing paragraph.
+
+With this update, this behavior has been corrected. Selecting a string and running `tinymce.activeEditor.execCommand('insertParagraph');` now replaces the selection with a paragraph break, as expected.
+
+
+=== Triple-clicking did not select the paragraph in Google Chrome in some circumstances
+
+Previously, when triple-clicking a text paragraph in a {productname} document that —
+
+* included a nested `contenteditable` structure below the paragraph;
+
+* had an already-selected paragraph above
+
+— the paragraph was not selected.
+
+Instead the insertion point moved to the beginning of the paragraph. This only occurred if the host browser was Google Chrome. When the host browser was, for example, Safari or Firefox, triple-clicking a paragraph in this circumstance selected the paragraph as expected.
+
+With this update, triple-clicking text a paragraph above a `contenteditable` structure and below an already selected paragraph selects the clicked-on paragraph, no matter the host browser.
+
+NOTE: this update works around a Google Chrome bug by implementing on-triple-click selection behavior within {productname} regardless of the host browser.
+
+
+=== When text within an inline boundary element was selected and the right-arrow key was pressed, the insertion point incorrectly moved to the left
+
+{productname} provides an `inline_boundaries_selector` option. If this option is loaded, by default, it sets `a[href]`,`code`, and `.mce-annotation` as the elements inline boundaries apply to. Other inline elements — such as `strong`, `em` and `span` — can be added to these defaults.
+
+Previously, however, when a custom `inline_boundaries_selector` was initialised in a {productname} document, an erroneous keyboard navigation behavior presented.
+
+For example, if the following was added to `tinymce.init` —
+
+[source,js]
+===
+tinymce.init({
+  selector: 'textarea',
+  inline_boundaries_selector: 'span'
+});
+===
```

Comment:
```suggestion
----
tinymce.init({
  selector: 'textarea', // change this value according to your HTML
  inline_boundaries_selector: 'span'
});
----
```
Just fixing some incorrect markup and the example didn't fully follow the style guide.